tests/test_cluster.py

```
@@ -170,29 +171,40 @@ def test_get_all_namespaces(self, mock_kube_contexts, mock_kube_config, mock_con
             assert len(namespaces) == 3
 
 
-class TestClusterGetters:
-    """Tests for getter methods."""
+class TestClusterProperties:
+    """Tests for property accessors."""
 
-    def test_get_controller_name(self, mock_kube_contexts, mock_kube_config, mock_controller, mock_namespaces):
-        """Test getting controller name."""
+    def test_controller_name(self, mock_kube_contexts, mock_kube_config, mock_controller, mock_namespaces):
```

Ruff (0.14.7)

Unused method argument: `mock_kube_contexts`

(ARG002)